participants/hacktoberfest.yml

```diff
@@ -2,9 +2,6 @@
 Name: DigitalOcean + Sponsors
 Website: https://hacktoberfest.com/
 Swag:
-  - stickers
-  - shirt
-  - trees
-Description: "Four pull requests to any participating public repo on GitHub or GitLab. Support open source and pick a limited edition T-shirt or plant a tree.  To reduce spam DigitalOcean introduced new measures on 2020: PRs count only if: **Submitted in a repo with the hacktoberfest topic AND during the month of October AND (The PR is merged OR The PR is labelled as hacktoberfest-accepted by a maintainer OR The PR has been approved)**"
+Description: "In this year, Hacktoberfest will be moving away from the official t-shirt rewards to a digital reward kit in pertnership with Holopin. Four pull requests to any participating public repo on GitHub or GitLab. To reduce spam DigitalOcean introduced new measures on 2020: PRs count only if: **Submitted in a repo with the hacktoberfest topic AND during the month of October AND (The PR is merged OR The PR is labelled as hacktoberfest-accepted by a maintainer OR The PR has been approved)**"
```

Comment:
As this repo probably spans throughout multiple years, could you please update "this year" to "from 2023 onwards" or so?